src/components/timecard/edit-shift-hours/EditShiftHours.jsx

```diff
@@ -140,16 +160,62 @@ const EditShiftHours = ({
           setShowEditShiftHours(false);
         }
       },
+      true,
       handleServerValidationErrors
     );
   };
 
+  const getCombinedErrors = () => {
+    const existingErrors =
+      Object.keys(errors).length > 0 ? errors : summaryErrors;
+
+    const combinedErrors = { ...existingErrors };
+
+    if (clashingProperty === 'startAndEndTime') {
+      combinedErrors['shift-start-time'] = {
+        message: (
+          <TimeInputErrors
+            clashingProperty={clashingProperty}
+            clashes={clashingTimes}
+          />
+        ),
+      };
```

Comment:
We could compress the code down a bit as this object is repeated 3 times, we could assign in to a variable before hand and then just assign it to these keys.

Reply:
👍  done

Reply:
Unresolving this as I've just changed it back; moving the object outside was raising an error because it was trying to create the object with no data when there were no time clash errors, and I can't think of a way to avoid it 🤔